### PR TITLE
Card review update

### PR DESCRIPTION
**Pull request by poojakelkar**
badge

- Change Card with badge
- and Card with dismiss

### CODE REVIEW

**Review by TejasShekar — APPROVED** [review]
Include the correct title in the meta tag.

File: styles.css [hunk 1]
```diff
@@ -257,44 +257,82 @@ body {
...
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
 }
-.card-dismiss-icom:hover {
-    color: var(--light-blue-color);
+.card-dismiss-title {
+}
+.card-subtitle {
+    font-size: 18px;
```
Comment (chanchal-panpaliya) [inline styles.css:334]: use rem or em unit for font-size .

File: styles.css [hunk 2]
```diff
@@ -356,8 +394,8 @@ body {
 /* horizontal card  */
 .card-img {
     position: relative;
-    width: 10rem;
-    height: 6rem;
+    width: 1005;
```
Comment (chanchal-panpaliya) [inline styles.css:397]: for width unit  use -  vh / vw units . avoid px.

**Review by chanchal-panpaliya — COMMENTED** [review]
code looks good . Need - units consistency .

File: styles.css (hunk 1 above)
Comment (poojakelkar) [reply styles.css:334]: ok will do that

File: styles.css (hunk 2 above)
Comment (poojakelkar) [reply styles.css:397]: ok

**Pull request merged**